tests/functional/ui/test_odf_storage_consumption_trend.py

```diff
@@ -0,0 +1,314 @@
+import logging
+import math
+import re
+import time
+
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    tier2,
+    ui,
+    ignore_leftovers,
+    black_squad,
+    skipif_ocp_version,
+    skipif_ocs_version,
+)
+from ocs_ci.helpers import helpers
+from ocs_ci.helpers.osd_resize import basic_resize_osd
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.cluster import get_used_and_total_capacity_in_gibibytes
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.pod import (
+    get_age_of_cluster_in_days,
+    get_mgr_pods,
+    get_ceph_tools_pod,
+    delete_pods,
+    get_prometheus_pods,
+)
+from ocs_ci.ocs.resources.storage_cluster import get_storage_size
+from ocs_ci.ocs.ui.validation_ui import ValidationUI
+
+logger = logging.getLogger(__name__)
+
+
+POD_OBJ = OCP(kind=constants.POD, namespace=config.ENV_DATA["cluster_namespace"])
+
+
+@tier2
+@ui
+@ignore_leftovers
```

Comment:
what are we ignoring here?

Reply:
osd resize is creating new rook-ceph-osd pods after deleting the exising pods. Due to this the test case is failing in teardown.